tests/local/delta/test_DeltaTableSpec.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+from textwrap import dedent
+
+from pyspark.sql import types as t
+
+from spetlr import Configurator
+from spetlr.deltaspec.DeltaTableSpec import DeltaTableSpec
+from tests.cluster.delta.deltaspec import tables
+
+
+class TestDeltaTableSpec(unittest.TestCase):
```

Comment:
do you test some of the places where you expect raises?

Reply:
I added some important tests.